include/sundials/sundials_matrix.h

```diff
@@ -155,19 +155,6 @@ SUNErrCode SUNMatMatvec(SUNMatrix A, N_Vector x, N_Vector y);
 SUNDIALS_EXPORT
 SUNErrCode SUNMatSpace(SUNMatrix A, long int* lenrw, long int* leniw);
 
-/*
- * -----------------------------------------------------------------
- * IV. SUNMatrix error codes
```

Comment:
Note removal in change log like in the solvers PR

Reply:
Will need to add to package change logs as well

Reply:
When everything is in staging, I will update the package docs with anything that didnt make it from the CHANGELOG.md otherwise itll create conflicts.